frontend/src/metabase/home/homepage/types.ts

```diff
@@ -0,0 +1,29 @@
+export interface User {
```

Comment:
As a note, in the `infra frontend` projects, when creating:
- an interface, why not call it that way? `IUser`, `IDatabase`, etc.
- a type, why not call it that way? `UserType`, `DatabaseType`, etc.

So it is clear what you are or want to use.

Reply:
This `I` convention seems to be not currently widespread, but that's worth discussion on FE meetings